packages/pockethost.io/src/components/ValidateUserToken.svelte

```diff
@@ -0,0 +1,39 @@
+<script>
+  import { client } from '$src/pocketbase'
+  import { globalUserData } from '$util/stores'
+  import { onMount } from 'svelte'
+  import { getRouter } from '$util/utilities'
+  import publicRoutes from '$util/public-routes.json'
+
+  const { refreshUserData } = client
+
+  onMount(async () => {
+    try {
+      // First check if the user even has a token
+      const doesUserTokenExist = localStorage.getItem('pocketbase_auth')
```

Comment:
Is there a way via the PocketBase JS client to determine this rather than going directly to localStorage? If a custom storage provider is ever used by the PB client, this code will become brittle.

Reply:
I'll check.

Reply:
Found it! I updated it to be `LocalAuthStore.clear();`

Reply:
Hmm yeah just in case I'm not understanding your intent... the pocketbase JS SDK has a pluggable storage provider framework so I don't think it's good to rely directly on any storage like `localStorage`. I assume there is a pocketbase SDK call you can make, like something in `client.authStore`, to see if the user has a token and is currently authenticated. 

If we assume a storage method, then we'll have to update this code if we ever swap providers. Not sure we ever would, it just feels a little brittle.

Reply:
https://github.com/pocketbase/js-sdk#authstore

`const doesUserTokenExist = !!client.authStore.token` should do it right?

Reply:
yeah, i removed all localstorage calls and instead are using PocketBase's built in getters. I haven't pushed the commits yet but will here in a minute. To get the token, I can call `getAuthStore.token` and then check if that exists or not. 

And then for clearing, I can just use the `logOut()` method and it clears storage. So if PocketBase ever moved to `sessionStorage` for instance, all our methods will work since they pull directly from the SDK.

Reply:
That's perfect man thank you. Sorry for doing so many laps on this!